doc/build-openbsd.md

```diff
@@ -46,15 +44,16 @@ from ports. However you can build it yourself, [using depends](/depends).
 Refer to [depends/README.md](/depends/README.md) for detailed instructions.
 
 ```bash
+pkg_add gmake gtar
 gmake -C depends NO_BOOST=1 NO_LIBEVENT=1 NO_QT=1 NO_SQLITE=1 NO_NATPMP=1 NO_UPNP=1 NO_ZMQ=1 NO_USDT=1
 ...
-to: /path/to/bitcoin/depends/x86_64-unknown-openbsd
+to: /path/to/bitcoin/depends/amd64-unknown-openbsd7.5
```

Comment:
Do we need to add the version suffixes here?

Reply:
We add nothing. It's `config.guess` output.

Reply:
> We add nothing.

? The diff is adding `7.5`. It's unnecessary, and just adds the need for future maintainence. It can be dropped.

Reply:
Sure, but it's still not necessarily correct or needed. i.e the current doc incorrectly assumes x86_64. This could just be `*-unknown-openbsd*`.

Reply:
Replaced with a placeholder.

Reply:
> Sure, but it's still not necessarily correct or needed. i.e the current doc incorrectly assumes x86_64. This could just be `*-unknown-openbsd*`.

Just noticed this comment. Implemented as suggested.